nebula-logger/tests/log-management/classes/LogMassDeleteExtension_Tests.cls

```diff
@@ -107,8 +107,19 @@ private class LogMassDeleteExtension_Tests {
         PageReference pageReference = Page.LogMassDelete;
         Test.setCurrentPage(pageReference);
 
-        User chatterUser = [SELECT Id FROM User WHERE Profile.Name = 'Chatter Free User' LIMIT 1];
-        System.runAs(chatterUser) {
+        Profile standardUserProfile = [SELECT Id FROM Profile WHERE Name = 'Standard User' LIMIT 1];
```

Comment:
thoughts on abstracting this out?

Reply:
It's been moved to a new class `LoggerTestUtils` - I should have created this class a long time ago, there's some other duplication in existing test classes that could/should be consolidated. I'll move more test methods to this class in a future PR.